### PR TITLE
finalized (hopefully) the navbar

### PR DESCRIPTION
**Pull request by Um-Brehon**
made the navbar shrink when clicking on it while it is extended. slowed down the transition. removed the yellow bars button decorations

### CODE REVIEW

**Review by Franck59840 — DISMISSED** [review]
good

File: src/Components/Navbar/index.jsx [hunk 1]
```diff
@@ -18,6 +18,7 @@ import './index.css';
 function NavBar() {
   const [sidebar, setSidebar] = useState(false);
   const showSidebar = () => setSidebar(!sidebar);
+  // const hideSidebar = () => setSidebar(sidebar);
```
Comment (LoicBrassart) [inline src/Components/Navbar/index.jsx]: Dead code

**Review by LoicBrassart — DISMISSED** [review]
Dead code to remove

**Comment by LoicBrassart** [comment]
Fix conflicts and reassign

**Review by Franck59840 — DISMISSED** [review]
Good delete dead code

**Comment by LoicBrassart** [comment]
Fix conflicts and reassign

**Review by Hacene010 — COMMENTED** [review]
No SM validation

**Comment by Um-Brehon** [comment]
but... he commented "good" 😭

**Comment by Hacene010** [comment]
> but... he commented "good" sob

Yes but you changed some things after his comment :). So he must validate them again

**Review by Franck59840 — APPROVED** [review]
Good

**Review by Hacene010 — APPROVED** [review]

**Review by Franck59840 — APPROVED** [review]

**Pull request merged**